RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -6,5 +6,148 @@
 
 import Foundation
 
-print("Hello, World!")
+var shouldRunGame = true
 
+var validInputRange: ClosedRange<Int> {
+    return 0...RockPaperScissors.allCases.count
+}
+
+enum Script: String, CustomStringConvertible {
+    var description: String {
+        self.rawValue
+    }
+    
+    case menu = "가위(1), 바위(2), 보(3)! <종료 : 0> : "
+    case win = "이겼습니다!"
+    case lose = "졌습니다!"
+    case draw = "비겼습니다!"
+    case gameEnd = "게임 종료"
+    case dismissedError = "잘못된 입력입니다. 다시 시도해주세요."
+    case choiceOutOfRangeError = "가위바위보와 매칭될 수 있는 유효값이 아닙니다."
+}
```

Comment:
말씀 드린 `CustomStringConvertible`을 사용해보셨네요! `rawValue`를 설정하시면 작성하신 형태로 사용하실 수 있고, 아래와 같은 형태로 사용하기도 합니다! extension을 사용한 이유는 프로토콜을 준수하기 위한 구현부를 명확히 나누어 가독성을 높이기 위함입니다 ([참고 링크](https://github.com/raywenderlich/swift-style-guide#protocol-conformance)).

```suggestion
enum Script {
    case menu
    case win
    case lose
    case draw
    case gameEnd
    case dismissedError
    case choiceOutOfRangeError
}

extension Script: CustomStringConvertible {
    var description: String {
        switch self {
        case .menu:
            return "가위(1), 바위(2), 보(3)! <종료 : 0> : "
        case .win:
            return "이겼습니다!"
        case .lose:
            return "졌습니다!"
        case .draw:
            return "비겼습니다!"
        case .gameEnd:
            return "게임 종료"
        case .dismissedError:
            return "잘못된 입력입니다. 다시 시도해주세요."
        case .choiceOutOfRangeError:
            return "가위바위보와 매칭될 수 있는 유효값이 아닙니다."
        }
    }
}
```